.ci/publish-releasenotes-githubpage.sh

```diff
@@ -15,10 +15,5 @@ CURRENT_VERSION=$(xmlstarlet sel -N pom=http://maven.apache.org/POM/4.0.0 \
                              -t -m pom:project -v pom:version pom.xml | sed "s/-SNAPSHOT//")
 echo CURRENT_VERSION="$CURRENT_VERSION"
```

Comment:
why we need this variable ?

Reply:
plase use https://github.com/checkstyle/checkstyle/issues/12128 as issue reference

Reply:
Changes Done